pkg/exporter/sumologicexporter/exporter_test.go

```diff
@@ -654,12 +627,12 @@ func TestPushJSONLogs_AttributeTranslation(t *testing.T) {
 				func(w http.ResponseWriter, req *http.Request) {
 					body := extractBody(t, req)
 					var regex string
-					regex += `{"host.type":"wizard","log":"Example log","timestamp":\d{13}}`
+					regex += `{"log":"Example log","timestamp":\d{13}}`
 					assert.Regexp(t, regex, body)
 
-					assert.Equal(t, "host.name=harry-potter", req.Header.Get("X-Sumo-Fields"))
-					assert.Equal(t, "harry-potter", req.Header.Get("X-Sumo-Category"))
-					assert.Equal(t, "undefined", req.Header.Get("X-Sumo-Host"))
+					assert.Equal(t, "host.name=harry-potter, host.type=wizard", req.Header.Get("X-Sumo-Fields"), "X-Sumo-Fields")
+					assert.Equal(t, "harry-potter", req.Header.Get("X-Sumo-Category"), "X-Sumo-Category")
+					assert.Equal(t, "undefined", req.Header.Get("X-Sumo-Host"), "X-Sumo-Host")
```

Comment:
I'm not sure if we need the third argument, if yes, we should use it for all comparisons in tests :shrug:

Reply:
The point of this is that we'll get this third argument in test failure output without the need to look at the code/line number.

I agree but at the same time I didn't want to change ALL TEST THE CODE. Perhaps an improvement for another time?

Reply:
> The point of this is that we'll get this third argument in test failure output without the need to look at the code/line number.

Most of times I need to do it anyway :sweat_smile: